README.md

```diff
@@ -8,3 +8,28 @@ An informative repo for all Code.gov repos
 To learn more about the project, check out this [blog post](https://www.whitehouse.gov/blog/2016/08/08/peoples-code).
 
 [Code.gov](https://code.gov) is an open source project, so we invite your contributions, be it in the form of code, design, or ideas.
+
+# Repositories
+
+As a project, [Code.gov](https://code.gov) is a collection of repositories that enable the collection of source code inventory files. All of our work is open source and we encorage you to take a look and, if you wish, contribute to our projects by submitting a pull request, Github Issue, or commenting on existing issues and pull requests.
+
+1. [code-gov-web](https://github.com/GSA/code-gov-web): Our frontend project. Currently deployed as a static site, this project consumes a series of JSON files to display project repositories, client side search index, and agency compliance dashboard.
+2. [code-gov-api](https://github.com/GSA/code-gov-web): Our backend API. An Express.js app backed by Elasticsearch. It's primary function is to index and make America's source code discoverable and searchable.
+3. [code-gov-harvester](https://github.com/GSA/code-gov-harvester): Our standalone source code inventory harvester. Currently this application will generate the files needed by our front-end to display project repositories and our client side search index. It also generates the data for the agency complaince daashboard.
```

Comment:
daashboard is spelled wrong, should be dashboard

Reply:
Under web -- hyphenate "client side" since it modifies index
Under API -- "It's primary" should be "its primary." 
Under harvester, 
   - omit "Currently". Currently is present tense but you use future tense later in the sentence. It should either be "Currently, this application generates" or "This application will generate."  
   - Take out the hyphen in "frontend" so you are consistent. 
   - Hyphenate "Client-side" as it modifies "index."